internal/diffmapper/diffmapper.go

```
@@ -55,7 +55,7 @@ func GetMap(diff string) ChangedLines {
 			// actually this is unchanged, but the reviewer may want to comment ot this as well
```

Returning latest element could give wrong mapping for out of bounds line number. Better to return an error.